cron/client.py

```
@@ -224,6 +224,12 @@ def do_measurement_control():
         # endlessly in validation until manually handled, which is what we want.
         time.sleep(config['cluster']['client']['time_between_control_workload_validations'])
 
+def reboot():
+    set_status('reboot')
+    subprocess.check_output(['sync'], encoding='UTF-8', errors='replace')
```

Ruff (0.15.15)

[error] Starting a process with a partial executable path

(S607)